src/host/stream.h

```diff
@@ -29,6 +29,12 @@ Revision History:
                                _Out_opt_ bool* const pPopupKeys,
                                _Out_opt_ DWORD* const pdwKeyState) noexcept;
 
+[[nodiscard]] NTSTATUS _ReadCharacterInput(InputBuffer& inputBuffer,

```

Comment:
nit: if this is going in the "public API" component of `stream.h`, it should not begin with a `_`